api/openstorage.yaml

```diff
@@ -774,17 +784,19 @@ paths:
                      "secure_device_path": "",
                      "background_processing": false
                     }]
-    post:
-      summary: Create New Volume
-      description: Create new volume according to spec
-      parameters:
-        - name: spec
-          in: body
+      post:
+        summary: Create New Volume
+        description: Create new volume according to spec
+        parameters:
+          - name: spec
+            in: body
             description: |
+             Volume create path example:
+             POST /v1/osd-volumes
               Volume spec example:
               ``` json
                   "locator": {
-                     "name": "volume1",
+                     "name": "myvolume",
                      "labels": "Initial"
```

Comment:
ok, i didn't get what labels are used for , but am assuming it for some QoS purposes or user defined values.
label: {SLA: "Gold"}